source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -46,6 +46,8 @@ constexpr auto DEVICE_DOES_NOT_SUPPORT_CDAQ_SYNC_CONNECTIONS_ERROR = -201450;
 constexpr auto EVERY_N_SAMPLES_EVENT_NOT_SUPPORTED_FOR_NON_BUFFERED_TASKS = -200848;
 constexpr auto EVERY_N_SAMPS_TRANSFERRED_FROM_BUFFER_EVENT_NOT_SUPPORTED_BY_DEVICE_ERROR = -200980;
 constexpr auto CANNOT_UNREGISTER_DAQMX_SOFTWARE_EVENT_WHILE_TASK_IS_RUNNING_ERROR = -200986;
+constexpr auto MEASUREMENT_TYPE_NOT_SUPPORTED_BY_SELECTED_PHYSICAL_CHANNEL_ERROR = -200431;
+constexpr auto THERMOCOUPLE_LEAD_OFFSET_NULLING_CALIBRATION_NOT_SUPPORTED_BY_DEVICE_ERROR = -201375;
```

Comment:
Consider matching the error constants in NIDAQmx.h (e.g. DAQmxErrorThrmcplLeadOffsetNullingCalNotSupported -> THRMCPL_LEAD_OFFSET_NULLING_CAL_NOT_SUPPORTED_ERROR)